pkg/kv/kvserver/batcheval/cmd_add_sstable.go

```diff
@@ -179,6 +182,7 @@ func EvalAddSSTable(
 	ms.Add(stats)
 
 	if args.IngestAsWrites {
+		span.RecordStructured(&types.StringValue{Value: fmt.Sprintf("ingesting SST (%d keys/%d bytes) via regular write batch", stats.KeyCount, len(args.Data))})
 		log.VEventf(ctx, 2, "ingesting SST (%d keys/%d bytes) via regular write batch", stats.KeyCount, len(args.Data))
```

Comment:
Could you remind me when this gets logged to the trace? Is there some redundancy between this line and the structured record recording above?

Reply:
It would only get logged to the span if the span is verbose, which is not the case by default for jobs. So under normal circumstances, we would not see the `log.VEventf` on the trace span.